client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -79,6 +80,10 @@ function PatternPreviewFragment( {
 	useTimeoutToResetBoolean( isPermalinkCopied, setIsPermalinkCopied );
 	useTimeoutToResetBoolean( isPatternCopied, setIsPatternCopied );
 
+	useEffect( () => {
+		ref.current?.dispatchEvent( new CustomEvent( 'patternPreviewResize', { bubbles: true } ) );
```

Comment:
Should we debounce/throttle it a little bit? Does it need to run so frequently?

Reply:
I've added some debouncing to the event listener. It might result in more "FOUC"-like moments, but it seems reasonable to reduce the number of times this runs.

Reply:
@fredrikekelund I am thinking, is it possible to keep it on wrapper instead of every element?
Since for example now, when we open `About` category, the event is triggered 92 times

Reply:
Oh, I see...